strategy/to_json/from_serializable.ts

```diff
@@ -8,6 +8,10 @@ export function fromSerializable(): ToJSONStrategy {
   return (value: Serializable | Serializable[]): JSONValue => {
     if (Array.isArray(value)) {
       return value.map((item) => item.tsSerialize());
+    } else if (value === null) {
+      // Objects can be null, return them without further serialization.
+      // Double cast required to return a JSONValue
+      return value as null as JSONValue;
     }
     return value.tsSerialize();
```

Comment:
Hmm, I don't think we need the `else if`, it could be another `if`, this is because the above `if` `return`'s and exits anyways so an `else` is not needed. I am also thinking about something like `return value ? value.tsSerialize() : value;`

what are your thoughts? Nice comments to btw!

Reply:
😅 

I changed to

```
    return value ? value.tsSerialize() : value as null as JSONValue;
```